Cataclysm/APLs/RogueCombat.simc

```diff
@@ -1,17 +1,20 @@
-Smufriks built form wowtbc logic
+Smufriks Combat
+
+actions+=/run_action_list,name=st
+actions+=/run_action_list,name=aoe,if=active_enemies>3
+
+actions.st+=/sinister_strike,if=combo_points<5|buff.slice_and_dice.down&combo_points=0
+actions.st+=/slice_and_dice,if=buff.slice_and_dice.remains<2|buff.slice_and_dice.down&combo_points>=1
+actions.st+=/killing_spree,if=energy<50&buff.deep_insight.up
+actions.st+=/adrenaline_rush,if=energy<20
+actions.st+=/rupture,if=combo_points=5&!ticking&(boss|active_enemies=1)&!buff.deep_insight.up&!buff.blade_flurry.up&!buff.adrenaline_rush.up&time_to_die>12&!buff.bloodlust.up
+actions.st+=/revealing_strike,if=combo_points<4&!buff.revealing_strike.up&buff.slice_and_dice.remains>5
+actions.st+=/eviscerate,if=combo_points=5&(buff.deep_insight.up|buff.shallow_insight.up)
+actions.st+=/gouge,if=combo_points<4&energy<40
 
-actions+=/killing_spree,if=energy<35&buff.adrenaline_rush.down
-actions+=/adrenaline_rush,if=energy<20
-actions+=/slice_and_dice,if=buff.slice_and_dice.remains<2
-actions+=/rupture,if=combo_points=5&!ticking&(boss|active_enemies=1)&!buff.deep_insight.up&!buff.blade_flurry.up&!buff.adrenaline_rush.up&time_to_die>12&!buff.bloodlust.up
-actions+=/revealing_strike,if=combo_points<4&!buff.revealing_strike.up&buff.slice_and_dice.remains>5
-actions+=/eviscerate,if=combo_points=5&(buff.deep_insight.up|buff.shallow_insight.up)
-actions+=/sinister_strike,if=buff.deep_insight.down&buff.shallow_insight.down
-actions+=/sinister_strike,if=combo_points<5
-actions+=/actions.aoe,if=active_enemies>3
 
 
 actions.aoe+=/blade_flurry,if=active_enemies>1&!buff.blade_flurry.up
 actions.aoe+=/fan_of_knives,if=active_enemies>3&energy<50&buff.deep_insight.up
 actions.aoe+=/actions
-actions.aoe+=/cancel_buff,name=blade_flurry,if=active_enemies=1&buff.blade_flurry.up
+actions.aoe+=/cancel_buff,name=blade_flurry,if=active_enemies=1&buff.blade_flurry.upactions+=/sequence,name=opening,if=buff.slice_and_dice.down&combo_points=0,actions=sinister_strike,slice_and_dice
```

Comment:
Sequences aren't supported adn there seems to be a missing line-break here.

Reply:
oh thats a typo by me